numpy/distutils/mingw32ccompiler.py

```diff
@@ -246,6 +250,32 @@ def object_filenames (self,
 
     # object_filenames ()
 
+    # What is the  policy for msvcrt / vcruntime? Python distributes it with
+    # the standard release. Is it appropriate to include python's install
+    # directory so that it is automatically found?
+    # The directory where python.exe is  located is in my search path, but
+    # that  did not fix this issue.
+    # I needed this change to have the config tests run. This links to the
+    # vcruntime in the python directory.
+    # Can this be fixed by an appropriate environment variable?
```

Comment:
Some of these comments feel like they belong in the PR, not the code (perhaps due to being first person)